docs/SequencesAndCollections.rst

```diff
@@ -55,11 +55,11 @@ represented by the `SequenceType` protocol::
 
   __ http://swiftdoc.org/v3.0/type/AnyIterator/
 
-As you can see, sequence does nothing more than deliver an iterator.
+As you can see, sequence does nothing more than delivering an iterator.
 To understand the need for iterators, it's important to distinguish
 the two kinds of sequences.
 
-* **Volatile** sequences like "stream of network packets," carry
+* **Volatile** sequences like "stream of network packets", carry
```

Comment:
Do we even need one here? Two is also fine, but a comma on just one end of the non-restrictive clause distorts the flow.

Reply:
@AnthonyLatsis Thank you again for your comment.
I just realized that I omitted comma before "like" in the process of submitting. I apologize for the confusion.  

I meant to suggest, ```"**Volatile** sequences, like "stream of network packets", carry``` as I thought line 62 was using same sentence structure as Line 63. 

May I ask for your opinion on this again?

Reply:
Sounds good to me! Either 2 commas or none is fine.

Reply:
@AnthonyLatsis  I added 2 commas as we discussed. Would you be able to call smoke test? Thank you and happy new year. :)